plat/common/Makefile.rules

```diff
@@ -17,3 +17,12 @@ endef
 define build_efi =
 	@$(SCRIPTS_DIR)/mkefi.py $(1)
 endef
+
+define get_symbol_addr =
```

Comment:
Is this used anywhere else now that you do the fetching in the script?

Reply:
No, I forgot to remove it, thanks 👍🏼